core/src/main/java/com/sequenceiq/cloudbreak/service/user/UserService.java

```diff
@@ -86,7 +88,7 @@ private User createUser(IdentityUser identityUser) {
                 workspace.setName(identityUser.getUsername());
                 workspace.setStatus(ACTIVE);
                 workspace.setDescription("Default workspace for the user.");
-                workspaceService.create(user, workspace);
+                workspaceService.create(user, workspace, ALL_READ, ALL_WRITE);
```

Comment:
This is not a solution for the problem statement.
It's wrong because two things:
1) Users should be able to invite others to their default workspaces. This is a feature that we already use on cloud 28.
2) The JIRA is about workspaces in general. If there is a workspace with only 1 manager, that manager shouldn't be able to kick himself or revoke his own manage rights.